string old_algo
string new_algo

---

uint8 OK = 0
uint8 ERROR = 1
uint8 result
